docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-infrastructure.adoc

```diff
@@ -26,26 +27,13 @@ The details and functionality listed below are a preview and the actual release
 |*Feature*
 |*Description*
 
-|*Login Experience Enhancements*
-//RLP-135039
-
-|Starting with the 24.5.1 release, users can expect a change in their login experience on the Prisma Cloud console. Updates to the Prisma Cloud login pages will provide support for all available authentication options (excluding IdP-initiated SAML SSO). This update also addresses an issue related to redirection to Palo Alto Networks SSO for login and addresses a bug preventing traversal of deep links for some users.
-
-*Impact*— The SSO Login Page will be updated to more clearly identify OIDC SSO login and add support for Palo Alto Networks SSO login. Redirection to Palo Alto Networks SSO will no longer occur when accessing Prisma Cloud links without a valid session. Deep links will be supported in cases in which previously they were not persisted through login flows.
-
-|*API Rate Limits*
-//RLP-129569
-
-|Prisma Cloud uses API https://pan.dev/prisma-cloud/api/cspm/rate-limits/[rate limits] at the endpoint level to protect the performance and availability of its services. Rate limits will be applied to additional APIs starting from the 24.5.2 release. Details of these rate limits will be made available in the supporting API documentation and Look Ahead Notices when these limits are introduced. 
```

Comment:
Updated Look ahead feedback. 
1. We would like to keep this section for consistency and add a link to the new table. (asciidoc snippet provided below)
_snippet_

```
|*API Rate Limits*
//RLP-129569

|Prisma Cloud uses API https://pan.dev/prisma-cloud/api/cspm/rate-limits/[rate limits] at the endpoint level to protect the performance and availability of its services. Rate limits will be applied to additional APIs starting from the 24.6.1 release. Details of these rate limits will be made available in the supporting API documentation and Look Ahead Notices when these limits are introduced.

For more information, see the <<api-rate-limits>> table.
```
![image](https://github.com/hlxsites/prisma-cloud-docs/assets/3386501/daf6794a-f328-44e5-ac97-d04b9970a848)